configs/components/ruby-shadow.rb

```diff
@@ -48,6 +48,7 @@
     # https://github.com/apalmblad/ruby-shadow/issues/26
     # if ruby-shadow gets a 3 release this should be removed
     pkg.apply_patch "#{base}/ruby-shadow-2.5.0-cflags.patch", strip: "0"
+    pkg.apply_patch "#{base}/ruby-shadow-taint.patch", strip: "0"
```

Comment:
I'm :+1: since this patch is to account for Ruby 3.2 as opposed to something specific to macOS. Could you submit that patch upstream also if you haven't already?

Also thanks for running the before and after. Pleasantly surprised the only failures are on platforms we haven't worked adding Ruby 3.2 support for.